docs/book/src/migration/multi-group.md

```diff
@@ -9,7 +9,7 @@ the Kubebuilder v2 scaffolding (as of Kubebuilder v2.0.0).
 
```

Comment:
Could we ensure that we explain all with go/v4, which is the current default scaffold and then, at the bottom, we create a new section for the go/v3 layout? Also, could we add a WARNING note saying the go/v3 is deprecated?